lib/game.rb

```diff
@@ -37,4 +39,53 @@ def self.lowest_total_score
       game.total_score
     end.min
   end
+
+  def self.game_count
+    GameFactory.all_games.length
+  end
+
+  def self.percentage_home_wins
+    (count_home_wins.to_f / game_count.to_f) * 100.00
```

Comment:
Awesome helpers, makes this really clean!

Reply:
Cool looks good, I guess some of these attributes will be removed per Kat's check-in suggestion of YAGNI (I'll have to do this for Teams too)